backend/start.sh

```
@@ -1,15 +1,42 @@
 #!/bin/bash
-# Start the FastAPI backend server (Modern UV Version)
+# Start the FastAPI backend server     echo "Starting FastAPI backend server (fallback uvicorn)..."
+    # Start uvicorn directly for consistency
+    cd "$SCRIPT_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)